test/e2e-cypress/tests/features/multiple-examples-core.js

```diff
@@ -0,0 +1,1311 @@
+/**
+ * @prettier
+ */
+
+describe("OpenAPI 3.0 Multiple Examples - core features", () => {
```

Comment:
TODO: break this gigantic file down into a file for each endpoint, plus a helpers file